README.md

```diff
@@ -1,2 +1,90 @@
 # Capstone-project
 capston project website
+## html and styles css & javascript
+
+- english language
+- visual code editor
+- html , css, javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://www.loom.com/share/404dfb258692414d9e5bccc10b82129f)
```

Comment:
- [ ] In your presentation you should highlight a piece of code that you find more interesting or you are proud of making.